### PR TITLE
[HttpFoundation] [Hackday] #9942 test: Request::getContent() for null value

### PR DESCRIPTION
**Pull request by mfoti**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #9942 |
| License | MIT |
| Doc PR |  |

Implementing test for issue #9942 to confirm that the bug is already fixed. It's possibile to close the issue. 

Thanks also to @baiolo, @andyroid1978 for contributions at HackDay of the SymfonyConf

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/Tests/RequestTest.php [hunk 1]
```diff
@@ -904,6 +904,13 @@ public function testGetContentReturnsResource()
         $this->assertTrue(feof($retval));
     }
 
+    public function testEmptyStringContentReturns()
+    {
+        $req = Request::create('test');
+
+        $this->assertEquals('', $req->getContent());
```
Comment (stof) [inline src/Symfony/Component/HttpFoundation/Tests/RequestTest.php]: Btw, with assertEquals, this test will pass for both null and empty string.

**Comment by fabpot** [comment]
Thank you @skler.

File: src/Symfony/Component/HttpFoundation/Tests/RequestTest.php (hunk 1 above)
Comment (fabpot) [inline src/Symfony/Component/HttpFoundation/Tests/RequestTest.php]: assertion fixed in b0a4c38

**Comment by fabpot** [comment]
reverted

**Pull request closed without merging**